collins/commands/util.go

```diff
@@ -69,6 +71,15 @@ func getCollinsClient(c *cli.Context) *collins.Client {
 	}
 
 	collins, err := collins.NewClientFromYaml()
+	if collins.Password == "" {
```

Comment:
What happens when this code is executed and piped from one collins command to the next?

Reply:
can you elaborate with an example use case?

Reply:
something like `collins query -n dnsnode | collins set -a test:value`

Reply:
it will fail in such a scenario.
i have updated the `README.md` with a note about this fail case